### PR TITLE
fixed symbolic link to custom cron.d directory

### PR DESCRIPTION
**Pull request by renskiy**

### CODE REVIEW

File: debian/start-cron [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 # make link to custom location of /etc/cron.d if provided
 if [ "${CRON_PATH}" ]; then
-    echo "CRON_PATH provided, creating link: /etc/cron.d -> ${CRON_PATH}"
```
Comment (alexvassel) [inline debian/start-cron:5]: why did you remove `echo`? was very useful

File: debian/start-cron [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
-    ln -sf "${CRON_PATH}" /etc/cron.d
+    rm -rf /etc/cron.d
```
Comment (alexvassel) [inline debian/start-cron:5]: why did you remove dir?

File: debian/start-cron (hunk 1 above)
Comment (renskiy) [reply debian/start-cron:5]: `ln -v` below does solve this issue

File: debian/start-cron (hunk 2 above)
Comment (renskiy) [reply debian/start-cron:5]: `ln -sf` creates symbolic link inside the directory causing cron jobs inside subdirectory of `/etc/cron.d`. Such case does not supported by `crond`. The only way to replace `/etc/cron.d` by a symbolic link to another directory is to remove it before.

**Review by alexvassel — APPROVED** [review]
